.travis.yml

```diff
@@ -15,6 +15,7 @@ addons:
 matrix:
   include:
   - rust: stable
+    name: Format
```

Comment:
it does not format but check the formatting

Reply:
So `Format check`?

Reply:
formatting check? or check of formatting?
format sounds like paper format

Reply:
Changed to `Formatting check`.